spec/spec_helper.rb

```diff
@@ -29,4 +29,7 @@
   # automatically. This will be the default behavior in future versions of
   # rspec-rails.
   config.infer_base_class_for_anonymous_controllers = false
+
+  config.include Capybara::DSL
```

Comment:
You could add a line like the following:
```ruby
RSpec.configuration.include Capybara::DSL, :type => :request
```

That has the advantage of only including the Capybara stuff for request specs (which integration specs also are), and not for everything.